suite-native/transactions/src/components/TransactionDetail/TransactionDetailHeader.tsx

```diff
@@ -3,52 +3,64 @@ import React from 'react';
 import { Box, DiscreetText, Text } from '@suite-native/atoms';
 import { Icon, IconName } from '@trezor/icons';
 import { TransactionType } from '@suite-common/wallet-types';
-import { prepareNativeStyle, useNativeStyles } from '@trezor/styles';
 import { useFormatters } from '@suite-common/formatters';
+import { Color } from '@trezor/theme';
 
 type TransactionDetailHeaderProps = {
     type: TransactionType;
     amount: string;
     fiatAmount: string | null;
 };
 
-const transactionTypeTextMap: Partial<Record<TransactionType, string>> = {
-    recv: 'received',
-    sent: 'sent',
+type TransactionTypeInfo = {
+    text: string;
+    iconName: IconName;
+    sign: string;
+    signColor: Color;
 };
 
-const transactionIconMap: Partial<Record<TransactionType, IconName>> = {
-    recv: 'receive',
-    sent: 'send',
+const transactionTypeInfo: Partial<Record<TransactionType, TransactionTypeInfo>> = {
+    recv: {
+        text: 'Received',
+        iconName: 'receive',
+        sign: '+',
+        signColor: 'forest',
+    },
+    sent: {
+        text: 'Sending',
+        iconName: 'send',
+        sign: '-',
+        signColor: 'red',
+    },
 };
 
-const transactionTypeStyle = prepareNativeStyle(utils => ({
-    ...utils.typography.titleSmall,
-    textTransform: 'capitalize',
-    color: utils.colors.forest,
-}));
-
 export const TransactionDetailHeader = ({
     type,
     amount,
     fiatAmount,
 }: TransactionDetailHeaderProps) => {
-    const { applyStyle } = useNativeStyles();
     const { FiatAmountFormatter } = useFormatters();
     return (
```

Comment:
We should have some fallback here for other than these two transaction types:
(and as bonus you can be first who will use satisfies operator in suite codebase)
```tsx
const transactionTypeInfo = {
    recv: {
        text: 'Received',
        iconName: 'receive',
        sign: '+',
        signColor: 'forest',
    },
    sent: {
        text: 'Sending',
        iconName: 'send',
        sign: '-',
        signColor: 'red',
    },
} as const satisfies Partial<Record<TransactionType, TransactionTypeInfo>>;

export const TransactionDetailHeader = ({
    type,
    amount,
    fiatAmount,
}: TransactionDetailHeaderProps) => {
    const { FiatAmountFormatter } = useFormatters();

    if (type !== 'recv' && type !== 'sent')
        return <ErrorMessage value={`Unknown transaction type: ${type}`} />;

    return (
        <Box alignItems="center">
            <Box flexDirection="row" alignItems="center" marginBottom="small">
                <Text variant="hint" color="gray600">
                    {transactionTypeInfo[type].text}
                </Text>
                <Icon
                    name={transactionTypeInfo[type]?.iconName ?? 'placeholder'}
                    color="gray600"
                    size="medium"
                />
            </Box>
            <Box flexDirection="row">
                <Text variant="titleMedium" color={transactionTypeInfo[type].signColor}>
                    {transactionTypeInfo[type].sign}
                </Text>
                <DiscreetText typography="titleMedium">{amount}</DiscreetText>
            </Box>
            <DiscreetText typography="label" color="gray700">
                {`≈  ${FiatAmountFormatter.format(fiatAmount ?? 0)}`}
            </DiscreetText>
        </Box>
    );
};
```

Reply:
https://github.com/trezor/trezor-suite/pull/7272/commits/1603f9f0ab3ce97771f9757144da5eda525746ab